src/sentry/models/options/user_option.py

```diff
@@ -220,4 +237,17 @@ def write_relocation_import(
         serializer_options = UserOptionsSerializer(data={self.key: self.value}, partial=True)
         serializer_options.is_valid(raise_exception=True)
 
+        # TODO(getsentry/team-ospo#190): Find a more general solution to one-off indices such as
```

Comment:
This issue is technically closed, but there are a number of still-active future improvements in the codebase marked with it, so I figured any grep that catches those will catch this one to when we decide to resolve them or point them to a different bug.